python/gym_jiminy/toolbox/gym_jiminy/toolbox/math/generic.py

```diff
@@ -74,3 +74,116 @@ def quat_to_rpy(quat: np.ndarray) -> np.ndarray:
     if quat.ndim == 1:
         return rpy[0]
     return rpy
+
+
+@nb.jit(nopython=True, nogil=True, inline='always')
+def quat_to_matrix(quat: np.ndarray) -> np.ndarray:
+    """Compute the Rotation Matrix representation of a single or a
+    batch of quaternions.
+
+    :param quat: 1D array, a 2D array or nD array whose first dimensions
+                 corresponds to the number of individual quaternions, and the
+                 last to the 4 coordinates [qx, qy, qz, qw].
+    """
+    qx, qy, qz, qw = np.atleast_2d(quat)[..., -4:].T
+    col_1 = np.stack(
```

Comment:
We don't follow black style in jiminy, in favor of something more compact (that is compliant with official Python formatting PEP):
```python
np.stack((
    qx * qx - qy * qy - qz * qz + qw * qw,
    2 * (qx * qy + qz * qw),
    2 * (qx * qz - qy * qw)
    ), -1)
```